skimage/future/graph/rag.py

```diff
@@ -413,14 +410,14 @@ def rag_boundary(labels, edge_map, connectivity=2):
     return rag
 
 
-def draw_rag(labels, rag, img, border_color=None, node_color='#ffff00',
-             edge_color='#00ff00', colormap=None, thresh=np.inf,
-             desaturate=False, in_place=True):
+def show_rag(labels, rag, img, border_color='black', edge_width=1.5,
+             edge_cmap='magma', img_cmap='bone', thresh=np.inf, in_place=True,
+             ax=None):
```

Comment:
Comment for future reference, you don't need to tackle in this PR, but raise an issue in that case: can borders and edges have alpha? That might make it even more visible.